src/components/DatePicker.js

```diff
@@ -36,7 +36,7 @@ function Picker({ value, minDate, maxDate, label, onChange, required, disabled }
                 autoOk
                 format="DD/MM/YYYY"
                 renderInput={(props) => <TextField {...props} />}
-                value={moment(value)}
+                value={value ? moment(value) : null}
```

Comment:
ah so it was trying to convert null to a moment date, half thought they'd have handled that